src/test/resources/render/sampleBase.html

```diff
@@ -0,0 +1,28 @@
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>ECharts Demo</title>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/echarts/5.2.2/echarts.min.js"
+        integrity="sha512-ivdGNkeO+FTZH5ZoVC4gS4ovGSiWc+6v60/hvHkccaMN2BXchfKdvEZtviy5L4xSpF8NPsfS0EVNSGf+EsUdxA=="
+        crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+    <style>
+        #display-container {
+            width: 100%;
+            height: 100%;
+        }
+    </style>
+</head>
+
+<body>
+    <div id="display-container">
+    </div>
+    <script type="text/javascript">
+        var chart = echarts.init(document.getElementById("display-container")); 
+        var option = {"grid":{"right":"4%","bottom":"3%","left":"3%","containLabel":true},"xAxis":{"type":"category","axisTick":{"alignWithLabel":true},"data":["Mon","Tue","Wed","Thu","Fri","Sat","Sun"]},"yAxis":{"type":"value"},"tooltip":{"axisPointer":{"type":"shadow"},"trigger":"axis"},"series":[{"type":"bar","name":"Direct","data":[10,52,200,334,390,330,220],"barWidth":"60%"}]}
```

Comment:
We can't assume the keys are always generated in the same order.

Reply:
Will the order change for the same option? If so, we may not be able to compare the option part between the generated html file and expected html file.

Reply:
We already have other unit tests for generated options. It's not necessary to test them here. 
BTW, I don't recommend to generate files in unit tests because the effects of unit tests shouldn't be persistent.
One approach is to mock the behavior of writing to files, but I don't think it's worth the time and effort.
I'm ok without unit tests on that.